### DISCUSSION TITLE
Welcome to qmgo Discussions!

### DISCUSSION THREAD

**Discussion by jiangz222** [qiniu/qmgo#140] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nanhai-13** [comment]


**Reply by cupen** [reply]
@LainNetWork  对赌一下？ 如果有下文，就请你帮忙提交 100 个 bugfix 或 feature improved.

btw: 这毕竟不是探索性质的东西，只是基于官方 go-driver 封装下，又是常用的基础设施，没道理烂尾。
除非哪天官方 go-driver 突然开窍整点给人用的 api，那样才有点可能。

**Reply by LainNetWork** [reply]
@cupen 你这个对赌也太狠了==，我也关注这个项目很久了，也有在issue提过建议和讨论，不过官方一直不是很活跃，之前说很快有生产的正式版发布，过了快半年也没动静，所以才这么吐槽了的。无意冒犯，之前的评论我自删了

**Reply by cupen** [reply]
@LainNetWork  现在有 v1.0.0 了。别删啊，打个一折只要 10 个总可以吧，参与参与就活跃了。
 我自己有个项目用的 mgo 也想迁过来。代码我看了，浅封装很 OK 妥妥的。

**Reply by nanhai-13** [reply]
前几天草率的发布所谓的1.0版本；感觉有点应付的态度；其实没有改什么实质性的东东；建议还是不要正式去用了；用MongoDB官方就是稍微需要专业一点而已！

**Reply by LainNetWork** [reply]
@cupen v1.0.0没有任何新增的代码，就是一个tag。你也别追着我要和我对赌，有这样虚空对赌的吗，你要和我对赌的话是不是也要出10个feature improved？一起来让这个项目变得更好用吧

**Reply by cupen** [reply]
@LainNetWork 我没有批评你的意思，只是指出你现在说这是 KPI 项目一定会黄这事对作者不公平，提对赌不过是帮作者挽回一点公平性。 另，我并没有追着你…… 路过罢了，别瞎说。

:smile: 至于你希望我也参与对赌，我感到荣幸。但是既然与你对赌，那得是发生相反的事情才能触发，就是：当被证明是 KPI 项目（迅速黄掉）的情况下，我就要…… 如何如何，对吧？ 没问题，到时就由我来维护，bugfix 我来摆平，我用的 feature 也由我来改善。这样有诚意吧？ 别客气，毕竟我也受益。

关于 v1.0.0 有木有实质性提交…… 封大版本一般是以功能点为界限的，功能定下了，没有新的 improved 或 bugfix，那么标一个 v1.0 表示稳定版是没问题的。唯一的问题只是 “ 哎呀，好大的 bug 居然没发现 ”，关于这个问题等大伙发现了啥牛X bug 再说吧…… 现在不用惊慌，有问题解决问题。

**Reply by LainNetWork** [reply]
@cupen 关于我之前的言论我在这里向本项目作者道歉，而且我之前也自删了，我当时每天都会来这个项目看一眼，因为我提的讨论作者一直没回，项目也没动静，看到这里有人吐槽就顺口一说了，实属无心之举。至于对赌就算了，没这个时间专门来做这个事情，平时遇到问题也本来就会发issue讨论的。

**Reply by jiangz222** [reply]
> 前几天草率的发布所谓的1.0版本；感觉有点应付的态度；其实没有改什么实质性的东东；建议还是不要正式去用了；用MongoDB官方就是稍微需要专业一点而已！

较早的时候，就已经实现了规划的v1需要的所有功能
v.1.0.0 在最近发布，主要在等我们线上稳定跑的时间够长，然后想看看社区是否有大的bug上报
当然大家的催更也有一定的关系，不过不是决定因素，也说不上因为这个仓促发布😄

**Reply by nanhai-13** [reply]
好的；那我有空时候；帮助外围生态圈建设尽点微薄之力😀
